assets/src/data/content/model/elements/factories.ts

```diff
@@ -73,6 +81,20 @@ export const Model = {
   calloutInline: (text = '') =>
     create<CalloutInline>({ type: 'callout_inline', children: [{ text }] }),
 
+  conjugationTable: () =>
+    Model.table([
```

Comment:
Can we change this so that the default conjugation table has four rows, and three columns, and reads:

```
<empty>   | Singular | Plural
1st Person | <empty> | <empty>
2nd Person | <empty> | <empty>
3rd Person | <empty> | <empty>
```

Reply:
New default: 
![image](https://user-images.githubusercontent.com/333265/192372360-e5e4dd5e-8189-4642-a6b8-402f35c9f3eb.png)